db/seeds.rb

```diff
@@ -278,46 +278,61 @@
 product_monitor.save!
 
 #Requests
-Request.create!(user_id: user_seeker_1.id, category_id: category_laptop.id)
-Request.create!(user_id: user_seeker_2.id, category_id: category_laptop.id, status: "Completed")
-Request.create!(user_id: user_seeker_5.id, category_id: category_mobile.id)
-Request.create!(user_id: user_seeker_7.id, category_id: category_printer.id)
-Request.create!(user_id: user_seeker_3.id, category_id: category_desktop.id)
-Request.create!(user_id: user_seeker_3.id, category_id: category_monitor.id)
-Request.create!(user_id: user_seeker_6.id, category_id: category_printer.id)
-Request.create!(user_id: user_seeker_4.id, category_id: category_camera.id)
-Request.create!(user_id: user_seeker_8.id, category_id: category_laptop.id)
-Request.create!(user_id: user_seeker_6.id, category_id: category_tablet.id, status: "Completed")
+Request.create!(user_id: user_seeker_1.id, category_id: category_laptop.id, description: "Amal needs a laptop to complete her university degree.")
+Request.create!(user_id: user_seeker_2.id, category_id: category_laptop.id, description: "Saloo needs a laptop to study Social Work.", status: "Completed")
+Request.create!(user_id: user_seeker_5.id, category_id: category_mobile.id, description: "Noor needs a mobile phone to start her own catering business.")
+Request.create!(user_id: user_seeker_7.id, category_id: category_printer.id, description: "Sulaiman needs a desktop computer and a printer to continue his job as an engineer")
+Request.create!(user_id: user_seeker_3.id, category_id: category_desktop.id, description: "Bob needs a desktop and monitor to continue his journey to be a software engineer.")
+Request.create!(user_id: user_seeker_3.id, category_id: category_monitor.id, description: "Bob needs a desktop and monitor to continue his journey to be a software engineer.")
```

Comment:
Can we make it so descriptions only refer to the request category, i.e. not have multiple categories in the description.
(1 request is for 1 category)